mesh/submesh/submesh_utils.cpp

```diff
@@ -117,18 +117,21 @@ void BuildVdofToVdofMap(const FiniteElementSpace& subfes,
 
             auto pm = parentfes.GetMesh();
 
+            const Geometry::Type face_geom =
+               pm->GetBdrElementBaseGeometry(parent_element_ids[i]);
```

Comment:
Similar to above, this call can also be replaced by a call to `GetBdrElementGeometry`.

Reply:
I went ahead and replaced all calls to `Mesh::GetBdrElementBaseGeometry` with `Mesh::GetBdrElementGeometry` in a67d9f915.